app/views/microposts/_micropost.html.erb

```diff
@@ -9,5 +9,7 @@
     <%= link_to "delete", micropost, method: :delete, data: { confirm: "You sure?" } %>
 
 </span>
+<%= render 'comments/comment', object: user %>
+<%= render 'likes/like_posts', object: user %>
```

Comment:
The content of the 2 partials are almost same.

Can we achieve this using single partial?

Reply:
i am trying but some error are given that why use two file